docs/application/web/guides/machine-learning/single.md

```diff
@@ -119,6 +119,39 @@ To invoke the neural network model, you need to create the `TensorsData` instanc
     var result = model.invoke(inputTensorsData);
     ```
 
+## Invoke asynchronously
+
+For inferences taking more time, it is recommended to use `invokeAsync()` method. It preserves responsive application UI during the inference:
+
+1. Create callbacks:
+
+    ```javascript
+    function errorCallback(error)
+    {
+      console.log("Error during invokeAsync: " + error.message);
+    }
+
+    function successCallback(result)
+    {
+      console.log("Inference finished successfully");
+
+      /* process result here */
+      console.log(result.getTensorRawData(0));
+
+      /* always call dispose() on no longer needed objects */
+      result.dispose();
+    }
+    ```
+
+2. Invoke the model asynchronously:
+
+    ```javascript
+    model.invokeAsync(inputTensorsData, successCallback, errorCallback);
+    ```
+
+    > [!NOTE]
+    > Ensure to dispose the `TensorsData` object provided by `successCallback`, when no longer needed.
+
```

Comment:
I reviewed this file again to check for any broken links in the document. I didn't find any broken links in this file.